calibration/OPTAAD/CGINS-OPTAAD-00123__20190528.csv

```diff
@@ -0,0 +1,9 @@
+ACS-123,CC_acwo,"[0.623141, 0.780460, 0.870543, 0.910777, 0.921968, 0.915975, 0.904084, 0.892356, 0.881589, 0.874823, 0.868734, 0.864407, 0.864121, 0.864883, 0.868693, 0.873932, 0.880231, 0.888067, 0.897466, 0.907215, 0.917392, 0.927768, 0.937908, 0.948165, 0.958695, 0.970901, 0.985081, 1.000177, 1.015521, 1.030061, 1.044937, 1.058112, 1.071373, 1.085530, 1.098844, 1.111894, 1.123067, 1.131672, 1.136586, 1.137393, 1.133648, 1.125431, 1.112766, 1.098945, 1.088041, 1.084680, 1.089048, 1.097481, 1.107045, 1.116609, 1.125679, 1.133717, 1.141057, 1.146102, 1.148213, 1.147791, 1.145252, 1.143126, 1.143171, 1.145791, 1.148280, 1.148117, 1.143751, 1.133525, 1.115987, 1.089807, 1.053917, 1.006504, 0.946358, 0.871370, 0.779971, 0.670303, 0.540676, 0.392148, 0.228013, 0.053210, -0.124136, -0.299418, -0.461744, -0.601443, -0.712146, -0.792515, -0.847014]",
+ACS-123,CC_cwlngth,"[399.6, 404.3, 408.2, 412.3, 416.2, 420.8, 425.8, 430.1, 434.3, 438.7, 443.2, 447.9, 452.9, 457.3, 461.8, 466.4, 471.3, 476.2, 481.5, 486.4, 490.7, 495.3, 500.1, 505.0, 510.1, 515.4, 520.6, 525.6, 530.3, 534.9, 539.6, 544.2, 549.2, 554.1, 559.1, 563.9, 568.5, 572.9, 577.1, 581.2, 585.2, 589.5, 593.9, 598.2, 602.9, 607.6, 612.3, 617.0, 621.5, 625.9, 630.0, 634.4, 638.5, 642.8, 647.3, 651.6, 656.0, 660.6, 664.9, 669.1, 673.4, 677.5, 681.4, 685.1, 688.9, 692.4, 696.0, 699.4, 702.8, 706.3, 709.5, 712.6, 716.0, 719.2, 722.2, 725.1, 728.1, 730.8, 733.4, 735.9, 738.9, 741.3, 743.7]",
+ACS-123,CC_ccwo,"[-0.498618, -0.344319, -0.212826, -0.093210, 0.007380, 0.095310, 0.173492, 0.244032, 0.308086, 0.375696, 0.402419, 0.468064, 0.508952, 0.546233, 0.581901, 0.615164, 0.647608, 0.678352, 0.703312, 0.729934, 0.754437, 0.776555, 0.799309, 0.819329, 0.836254, 0.852307, 0.867960, 0.882416, 0.897542, 0.916743, 0.914853, 0.932564, 0.949032, 0.957329, 0.969123, 0.980420, 0.988142, 0.994238, 0.996334, 0.994933, 0.989130, 0.978213, 0.961871, 0.941630, 0.922746, 0.911278, 0.906563, 0.908679, 0.912231, 0.915034, 0.917930, 0.919936, 0.921579, 0.921040, 0.917760, 0.911510, 0.904073, 0.895753, 0.888826, 0.887002, 0.885137, 0.881727, 0.876172, 0.863632, 0.844927, 0.818132, 0.781609, 0.735241, 0.677348, 0.603967, 0.517919, 0.412230, 0.286593, 0.143132, -0.020016, -0.193495, -0.374440, -0.556447, -0.726246, -0.880431, -1.008006, -1.101108, -1.167637]",
+ACS-123,CC_tbins,"[0.881507, 1.276132, 2.461481, 3.479355, 4.496000, 5.520000, 6.510833, 7.484348, 8.483200, 9.511667, 10.522000, 11.499583, 12.509167, 13.480000, 14.500000, 15.488400, 16.505200, 17.492000, 18.508696, 19.495200, 20.497692, 21.499200, 22.496800, 23.520400, 24.482400, 25.524000, 26.494167, 27.506400, 28.515200, 29.494583, 30.511481, 31.500000, 32.477692, 33.522308, 34.418621]",
+ACS-123,CC_taarray,SheetRef:CC_taarray,
+ACS-123,CC_tcarray,SheetRef:CC_tcarray,
```

Comment:
Do we verify these with a script?

Reply:
I spot check them with acs123.dev (opened in a text editor).  Looks good.